src/modules/generateRandomNumber.js

```diff
@@ -8,6 +8,19 @@
  */
 function generateRandomNumber() {
   /* Write your code here */
+  const availableNums = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const result = [];
+
+  const getRandomN = () => Math.floor(Math.random() * availableNums.length);
+
+  result.push(availableNums.splice(getRandomN(), 1)[0]);
+  availableNums.unshift(0);
+
+  while (result.length < 4) {
+    result.push(availableNums.splice(getRandomN(), 1)[0]);
+  }
+
+  return Number(result.join(''));
```

Comment:
Converting an array of numbers to a string and then back to a number can be unnecessary and may lead to confusion. Consider returning the array itself or joining the array elements into a string and returning that string.